dependencies.yaml

```
@@ -39,6 +54,22 @@ dependencies:
           - matrix:
             packages:
               - python>=3.10,<3.14
+  run_python:
+    common:
+      - output_types: [conda, requirements, pyproject]
+        packages:
+          - nvidia-ml-py>=12.0
+          - packaging
+          - psutil
+          - rich
+          - rich-click
+      - output_types: [conda]
+        packages:
+          - importlib-metadata >=4.13.0
+      # "python_version" specifier is a 'pip'-specific thing, so this needs its own group
+      - output_types: [pyproject, requirements]
+        packages:
+          - importlib_metadata >= 4.13.0; python_version < '3.12'
```

```suggestion
          - importlib-metadata >= 4.13.0; python_version < '3.12'
```

oops, I meant to change this one from `_` to `-` too